packages/slice-machine/src/features/customTypes/customTypesBuilder/SliceZoneBlankSlate.tsx

```diff
@@ -34,58 +35,62 @@ export const SliceZoneBlankSlate: FC<SliceZoneBlankSlateProps> = ({
     <Box
       flexGrow={1}
       justifyContent="center"
+      alignItems="center"
+      padding={32}
       data-testid="slice-zone-blank-slate"
     >
-      <BlankSlate backgroundImage="/blank-slate-slice-zone.png">
-        <BlankSlateContent>
-          <Box justifyContent="center" padding={{ bottom: 16 }}>
-            <SliceMachinePrinterIcon />
-          </Box>
-          <BlankSlateTitle>Add slices</BlankSlateTitle>
-          <BlankSlateDescription>
-            Slices are website sections that you can reuse on different pages
-            with different content. Each slice has its own component in your
-            code.
-          </BlankSlateDescription>
-          <BlankSlateActions>
-            <ActionList>
-              {aiSliceGenerationExperiment.eligible && (
-                <ActionListItem
-                  startIcon="autoFixHigh"
-                  onClick={openGenerateSliceWithAiModal}
-                  description="Let AI instantly create a Slice for you."
-                >
-                  Generate with AI
-                </ActionListItem>
-              )}
+      <BlankSlate>
+        <Box justifyContent="center" padding={{ bottom: 16 }}>
```

Comment:
do we need this box? how does it look without the artificial bottom padding? it'd be great if we can avoid diverging from the original blank slate, seems we're pretty close

Reply:
No, we don't need this anymore. It's removed ✔️